config/admin-functions/hash-signature.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+echo "Retrieving image signature..."
```

Comment:
Should we say that this might take some time?

Reply:
Good call.